mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -171,19 +177,39 @@ class ExperimentView extends Component {
         sortValue = runInfo[sort.key];
       }
 
+      const checkboxHandler = () => this.onCheckbox(runInfo.run_uuid);
+      let rowContents;
+      if (this.state.showMultiColumns) {
+        rowContents = ExperimentView.runInfoToRow({
+          runInfo: runInfo,
+          checkboxHandler: checkboxHandler,
+          paramKeyList: paramKeyList,
+          metricKeyList: metricKeyList,
+          paramsMap: paramsMap,
+          metricsMap: metricsMap,
+          tags: this.props.tagsList[idx],
+          metricRanges: metricRanges,
+          selected: !!this.state.runsSelected[runInfo.run_uuid]});
+      } else {
+        rowContents = ExperimentView.runInfoToRowCompact({
+          runInfo: runInfo,
+          checkboxHandler: checkboxHandler,
+          paramKeyList: paramKeyList,
+          metricKeyList: metricKeyList,
+          setSortBy: this.setSortBy,
+          hoverState: this.state.hoverState,
+          sortState: this.state.sort,
+          onHover: this.onHover.bind(this),
```

Comment:
Can the hover state be pushed down into the ExperimentCompactView component?

Reply:
Yeah definitely makes sense for it to be in `ExperimentRunsTableCompact`, the only tricky thing is that `hoverState` / `onHover` are used exclusively in the rows of the compact table, so I'd have to push the row-rendering logic into `ExperimentRunsTableCompact` (which is where it should ultimately live anyways). Happy to do it if it simplifies #567, let me know either way!